### PR TITLE
[ProfileData] add exchange_type

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_commons/profiles/profile_data.py [hunk 1]
```diff
@@ -35,15 +35,18 @@ class ProfileDetailsData(octobot_commons.dataclasses.FlexibleDataclass):
 class CryptoCurrencyData(octobot_commons.dataclasses.FlexibleDataclass):
     trading_pairs: list[str]
     name: typing.Union[str, None] = None
+    leverage: typing.Union[float, None] = None
```
Comment (GuillaumeDSM) [inline octobot_commons/profiles/profile_data.py:38]: this is to set leverage for a specific pair

File: octobot_commons/profiles/profile_data.py [hunk 2]
```diff
@@ -35,15 +35,18 @@ class ProfileDetailsData(octobot_commons.dataclasses.FlexibleDataclass):
...
 
 
 @dataclasses.dataclass
 class ExchangeData(octobot_commons.dataclasses.FlexibleDataclass):
     exchange_credential_id: typing.Union[str, None] = None
     internal_name: typing.Union[str, None] = None
+    exchange_type: str = constants.DEFAULT_EXCHANGE_TYPE
     exchange_id: typing.Union[str, None] = None
     proxy_id: typing.Union[str, None] = None
+    default_leverage: typing.Union[float, None] = None
```
Comment (GuillaumeDSM) [inline octobot_commons/profiles/profile_data.py:49]: this is to set a default leverage for a bot

Comment (Herklos) [reply octobot_commons/profiles/profile_data.py:49]: Should we create a FutureExchangeData and put default leverage in it? As, if I remember well, margin trading also has a "leverage" value

File: octobot_commons/profiles/profile_data.py (hunk 1 above)
Comment (Herklos) [reply octobot_commons/profiles/profile_data.py:38]: (same as the other comment)

File: octobot_commons/profiles/profile_data.py (hunk 2 above)
Comment (GuillaumeDSM) [reply octobot_commons/profiles/profile_data.py:58]: Up !

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Pull request merged**